experimental/algorithm/LAGr_Modularity.c

```diff
@@ -95,43 +149,43 @@ int LAGr_Modularity(
     }
     if (nclusters != nk_out)
     {
-        GRB_TRY(GrB_assign(k_out, k_out, NULL, vmask, GrB_ALL, nclusters, GrB_DESC_SC));
+        GRB_TRY(GrB_assign(k_out, k_out, NULL, vmask, GrB_ALL, nclusters,
+                           GrB_DESC_SC));
     }
     if (nclusters != nk_in)
     {
-        GRB_TRY(GrB_assign(k_in, k_in, NULL, vmask, GrB_ALL, nclusters, GrB_DESC_SC));
+        GRB_TRY(GrB_assign(k_in, k_in, NULL, vmask, GrB_ALL, nclusters,
+                           GrB_DESC_SC));
     }
 
     // Extract actual values of l, k_out, and k_in for modularity calculations
-    GrB_Index *lX, *k_outX, *k_inX;
-    LAGRAPH_TRY(LAGraph_Malloc((void **)&lX, nclusters, sizeof(GrB_Index), msg));
-    LAGRAPH_TRY(LAGraph_Malloc((void **)&k_outX, nclusters, sizeof(GrB_Index), msg));
-    LAGRAPH_TRY(LAGraph_Malloc((void **)&k_inX, nclusters, sizeof(GrB_Index), msg));
+    LAGRAPH_TRY(
+        LAGraph_Malloc((void **)&lX, nclusters, sizeof(GrB_Index), msg));
+    LAGRAPH_TRY(
+        LAGraph_Malloc((void **)&k_outX, nclusters, sizeof(GrB_Index), msg));
+    LAGRAPH_TRY(
+        LAGraph_Malloc((void **)&k_inX, nclusters, sizeof(GrB_Index), msg));
     GRB_TRY(GrB_Vector_extractTuples_INT64(NULL, lX, &nclusters, l));
     GRB_TRY(GrB_Vector_extractTuples_INT64(NULL, k_outX, &nclusters, k_out));
     GRB_TRY(GrB_Vector_extractTuples_INT64(NULL, k_inX, &nclusters, k_in));
 
     GrB_Index m, out_degree_sum, in_degree_sum, L_c;
-    GRB_TRY(GrB_reduce(&out_degree_sum, NULL, GrB_PLUS_MONOID_INT64, out_degree, NULL));
+    GRB_TRY(GrB_reduce(&out_degree_sum, NULL, GrB_PLUS_MONOID_INT64, out_degree,
+                       NULL));
 
     m = out_degree_sum;
     double norm = 1.0 / (m * m);
 
-    // Compute modularity
+    // compute modularity
```

Comment:
Need input on if there needs to be a distinction between undirected and directed. I feel like there should be, however I get consistent results with this code.